### ISSUE TITLE
False negative in the expat-xxe detection rule

### ISSUE THREAD

**Issue by 9iang22** [returntocorp/sgrep-rules#3724]
**Describe the bug**
This is about an FN in the [expat-xxe rule](https://github.com/semgrep/semgrep-rules/blob/develop/javascript/node-expat/security/audit/expat-xxe.yaml)

If we use `import` to import the expat module,  the rule will miss the case.

For example.

```
import * as expat from 'node-expat';

function test(input) {
    const parser = new expat.Parser();
    parser.parse(input);
}
```

**Priority**
How important is this to you?
- [ ] P0: blocking me from making progress
- [x] P1: this will block me in the near future
- [ ] P2: annoying but not blocking me
